File: Assets/Plugins/Android/BLEDebug.kt
package org.echelon.bleplugin

import android.util.Log

object BLEDebug {
    private const val enable = true
    const val tag = "UnityNativeBLE"

    fun logInfo(message: String) {
        if (enable) {
            Log.i(tag, message)
        }
    }

    fun logWarning(message: String) {
        Log.w(tag, message)
    }

    fun logError(message: String) {
        Log.e(tag, message)
    }
}

File: Assets/Plugins/BLE/BLEDebug.cs
using UnityEngine;

public static class BLEDebug
{
    public static void LogInfo(string message)
    {
        //if (Debug.isDebugBuild)
        //{
        Debug.Log(message);
        //}
    }
    public static void LogWarning(string message)
    {
        //if (Debug.isDebugBuild)
        //{
        Debug.LogWarning(message);
        //}
    }
    public static void LogError(string message)
    {
        Debug.LogError(message);
    }
}